templates/scenarios/csharp/command-and-response/teamsapp.yml.tpl

```diff
@@ -20,15 +20,30 @@ provision:
       botId: BOT_ID # The Azure Active Directory application's client id created for bot.
       botPassword: SECRET_BOT_PASSWORD # The Azure Active Directory application's client secret created for bot. 
 
-  - uses: arm/deploy # Deploy given ARM templates parallelly.
+  - uses: arm/deploy  # Deploy given ARM templates parallelly.
     with:
-      subscriptionId: ${{AZURE_SUBSCRIPTION_ID}} # The AZURE_SUBSCRIPTION_ID is a built-in environment variable. TeamsFx will ask you select one subscription if its value is empty. You're free to reference other environment varialbe here, but TeamsFx will not ask you to select subscription if it's empty in this case.
-      resourceGroupName: ${{AZURE_RESOURCE_GROUP_NAME}} # The AZURE_RESOURCE_GROUP_NAME is a built-in environment variable. TeamsFx will ask you to select or create one resource group if its value is empty. You're free to reference other environment varialbe here, but TeamsFx will not ask you to select or create resource grouop if it's empty in this case.
+      # AZURE_SUBSCRIPTION_ID is a built-in environment variable,
+      # if its value is empty, TeamsFx will prompt you to select a subscription.
+      # Referencing other environment variables with empty values
+      # will skip the subscription selection prompt.
+      subscriptionId: ${{AZURE_SUBSCRIPTION_ID}}
+      # AZURE_SUBSCRIPTION_ID is a built-in environment variable,
```

Comment:
`AZURE_RESOURCE_GROUP_NAME` here? Apply to all other files as well.

Reply:
Thanks. It is fixed.